### PR TITLE
Adds 14 colorful beanies to game/clothesmat

### PR DESCRIPTION
**Pull request by Niknakflak**
Sprites by Nienhaus with edits done to them by me
image
The colors look almost identical but now only the white, and striped versions of the beanies exist with the other colors being done via color variable.  The picture is left as is but remember that all the solid color icons don't exist anymore.
- Adds 14 colorful beanies to the game, available in the clothesmat vending machine

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1:

Greyscale the hats and color them via code. Icon bloat.

**Comment by Niknakflak** [comment]
It's the same as the scarves, consider offing yourself instead.

**Comment by jughu** [comment]
Add a red white striped one for waldo memes

**Comment by xxalpha** [comment]
grayscale

**Comment by yackemflam** [comment]
Grayscale
no more bloat
The game is getting too large

**Comment by TechnoAlchemisto** [comment]
CL?

**Comment by optimumtact** [comment]
First, please don't tell people to kill themself for offering an opinon, it's hardly reasonable.

Secondly, they're right, implement these as greyscale sprites and use the byond colouring features.

Just because features introduced before this functionality was available don't use it doesn't give you a right to not use it.

**Comment by Niknakflak** [comment]
changes made

File: code/modules/clothing/head/beanie.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+/obj/item/clothing/head/beanie/orange
+	name = "orange beanie"
+	icon_state = "beanie"
+	color = "#C67A4B" //orange
+	item_color = "orange"
+
+/obj/item/clothing/head/beanie/cyan
+	name = "cyan beanie"
+	icon_state = "beanie"
+	color = #54A3CE //Cyan (Or close to it)
```
Comment (optimumtact) [inline code/modules/clothing/head/beanie.dm]: you forgot to add " " around this

**Comment by Iamgoofball** [comment]
:+1: beanies are cool

File: code/modules/clothing/head/beanie.dm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	color = "#C67A4B" //orange
+	item_color = "orange"
+
+/obj/item/clothing/head/beanie/cyan
+	name = "cyan beanie"
+	icon_state = "beanie"
+	color = "#54A3CE" //Cyan (Or close to it)
+	item_color = "cyan"
+
+//Striped Beanies have unique sprites
```
Comment (Iamgoofball) [inline code/modules/clothing/head/beanie.dm]: You COULD in theory do these colored with overlays, but that's a lot of work so I'm not gonna pester you about this one.

**Comment by Niknakflak** [comment]
too bad you aren't
#REKT

**Comment by optimumtact** [comment]
Thank

**Pull request merged**